test.sh

```diff
@@ -0,0 +1,6 @@
+namespace=CtfPerf
+
+for filename in bin/*
+do
+    node $filename
```

Comment:
how about running the `aws cloudwatch put-metrics` command with a `--dry-run` flag?

Reply:
That will hide the exit codes caused by the scripts generating the output. The tests better just output the diff, so we can see them, and get notified if they fail.